doc/proxy-metrics.md

```diff
@@ -138,35 +138,23 @@ request_total{
 The following metrics are collected at the level of the underlying transport
 layer.
 
-### `tcp_accept_open_total`
+### `tcp_open_total`
 
-A counter of the total number of transport connections which have been accepted
-by the proxy.
+A counter of the total number of opened transport connections.
 
-### `tcp_accept_close_total`
+### `tcp_close_total`
 
-A counter of the total number of transport connections accepted by the proxy
-which have been closed.
+A counter of the total number of transport connections which have closed.
 
-### `tcp_connect_open_total`
-
-A counter of the total number of transport connections which have been opened
-by the proxy.
-
-### `tcp_connect_close_total`
-
-A counter of the total number of transport connections opened by the proxy
-which have been closed.
-
-### `tcp_connections_open`
+### `tcp_open_connections`
 
 A gauge of the number of transport connections currently open.
 
-### `sent_bytes`
+### `tcp_write_bytes_total`
 
 A counter of the total number of sent bytes.
 
-### `received_bytes`
+### `tcp_read_bytes_total`
 
 A counter of the total number of recieved bytes.
```

Comment:
We should note that these are currently incremented only when the connection completes.